jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -120,9 +125,22 @@ public JvmServiceImpl(final JvmPersistenceService jvmPersistenceService,
 
 
     protected Jvm createJvm(final CreateJvmRequest aCreateJvmRequest) {
+        validateWebserverNameConflict(aCreateJvmRequest);
         return jvmPersistenceService.createJvm(aCreateJvmRequest);
     }
 
+    private void validateWebserverNameConflict(CreateJvmRequest aCreateJvmRequest) {
+        try {
+            if (null != webServerPersistenceService.findWebServerByName(aCreateJvmRequest.getJvmName())) {

```

Comment:
Remove the "if != null" around the call to findWebServerByName. Since the API is checking for an exact name change, if the NoResultException isn't thrown then there was a conflict.